radio/src/targets/nv14/libopenui_config.h

```diff
@@ -25,7 +25,7 @@
 #include "libopenui_defines.h"
 
 #include "colors.h"
-#include "board.h"
+#include "hal.h"
```

Comment:
What do we need exactly from `hal.h`? It would probably make sense to only these defined somewhere, instead of using the full `hal.h`.

Reply:
The LCD screen size